src/pal/src/include/pal/module.h

```diff
@@ -31,13 +31,14 @@ typedef VOID (PALAPI *PUNREGISTER_MODULE)(HINSTANCE);           /* used to clean
 
 typedef struct _MODSTRUCT
 {
-    HMODULE self;           /* circular reference to this module */
-    void *dl_handle;        /* handle returned by dlopen() */
-    HINSTANCE hinstance;    /* handle returned by PAL_RegisterLibrary */
-    LPWSTR lib_name;        /* full path of module */
-    INT refcount;           /* reference count */
-                            /* -1 means infinite reference count - module is never released */
-    BOOL threadLibCalls;    /* TRUE for DLL_THREAD_ATTACH/DETACH notifications enabled, FALSE if they are disabled */
+    HMODULE self;                   /* circular reference to this module */
+    void *dl_handle;                /* handle returned by dlopen() */
+    HINSTANCE hinstance;            /* handle returned by PAL_RegisterLibrary */
+    LPWSTR lib_name;                /* full path of module */
+    INT refcount;                   /* reference count */
+                                    /* -1 means infinite reference count - module is never released */
+    INT refcount_get_native_handle; /* number of calls to PAL_GetLibraryNativeHandle */
```

Comment:
This new field needs to be initialized to zero in `LOADAllocModule` and `LOADInitializeModules`

Reply:
I added the initialization